stl/inc/functional

```diff
@@ -1247,6 +1251,10 @@ union alignas(max_align_t) _Move_only_function_data {
 // Treat a small function as if it has this size too if it fits and is trivially copyable.
 inline constexpr size_t _Minimum_function_size = 2 * sizeof(void*);
 
+// The below functions are __stdcall as they are called by pointers from _Move_only_function_base::_Impl_t.
+// (We use explicit __stdcall to make the ABI stable for translation units with different calling convention options.)

```

Comment:
stdcall is not the default calling convention...

Reply:
That's correct - it's not the default calling convention, we just need to choose a fixed calling convention that isn't sensitive to `/Gd /Gr /Gz /Gv`.